stagecoach/monitors/balance/app.py

```diff
@@ -44,34 +43,31 @@ async def _handler(publishers=None, threshold_wei=0.1 * 10**18):
 
         addresses.add(address)
 
-        # Set publisher private key to None because we aren't using the client for protected invokes
         publisher_client = EmpiricPublisherClient()
         balance = await publisher_client.get_balance(address)
 
         if balance < threshold_wei:
-            logger.warn(
-                f"\nWarning: Balance below threshold! Publisher: {felt_to_str(publisher)}, address: {address}, balance in ETH: {balance/(10**18)}\n"
+            logger.warning(
+                f"Balance below threshold for publisher: {felt_to_str(publisher)}, address: {hex(address)}, balance in ETH: {balance/(10**18)}"
             )
             all_above_threshold = False
             requests.post(
                 slack_url,
                 headers={"Authorization": f"Bearer {slack_bot_oauth_token}"},
                 data={
-                    "text": f"Balance below threshold! Publisher: {felt_to_str(publisher)}, address: {address}, balance in ETH: {balance/(10**18)}",
+                    "text": f"Balance below threshold for publisher: {felt_to_str(publisher)}, address: {hex(address)}, balance in ETH: {balance/(10**18)}",
```

Comment:
why not transfer funds to the wallet in this case?

Reply:
```
client = ... # account client
contract = Contract(client=client, address=0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7, abi=ERC20_ABI)
await client.transfer(recipient, int(1e18))
```

Reply:
this would transfer one ETH, and then you could just provide a list of addresses to check for balance

Reply:
Will do in a future PR: https://github.com/42labs/Empiric/issues/238